content/guides/authentication/sso.md

```diff
@@ -21,13 +21,12 @@ being built.
 
 ## Custom Apps with Client-side Authentication
 
-When using an application configured as a [Custom App][custom_app] that uses
+When using a [Custom App][custom_app] that uses
 client-side [OAuth 2.0] to authenticate users, the Box API will automatically
-redirect a user's browser to the configured SSO log-in screen.
+redirect a user's browser to their enterprise's configured SSO log-in screen.
 
-If the enterprise is configured to require SSO, rather than just enabled, then
-every user that wants to log in  must have a SSO user profile associated their
-Box account.
+Users who are a part of enterprises that require SSO, versus only enabling  
+SSO, must have an SSO user profile associated with their Box account.
```

Comment:
I don't actually get what this means. What errors might occur here? Who should care about this? Who could fix this?

Reply:
It's not so much an explicit error, it's more just what happens if you're trying to log in to an SSO provider (e.g. Okta) but you don't have an account on the Okta side. Developers would care about this since it's somewhat common for people to use OAuth2 and then have just one managed user as a "service account". To demonstrate who could fix it, I think it the best way would be to make it clear where the issue will occur - at the SSO provider's login page. Added 

"In an SSO Required state, Box enforces redirection to the user's SSO provider's
authentication flow. Thus, any Box users who are a part of an enterprise that is
SSO required must have a profile with the SSO provider, or else they will be
unable to authenticate while at the SSO provider's login screen. This is most 
relevant for applications who utilize managed users that are only used for 
platform use cases."

To address this